test/utils/integration_test_utils.py

```diff
@@ -99,3 +99,12 @@ def assert_connection_exists(
         """
     ).fetchall()
     assert result
+
+
+def set_language_definition(language_alias: str, pyexasol_connection: ExaConnection
```

Comment:
I think, it would be more robust, if we define this as context manager and reset the language definitions at the end